example/dd4tmp/subscriber.json

```diff
@@ -1,10 +1,12 @@
-{
-    "first_name": "Logan",
-    "plan_type": "GOLD",
-    "Member_Busname": "Makana Health",
-    "Member_Addr": "1 Salesforce Way",
-    "Member_City": "San Francisco",
-    "Member_State": "CA",
-    "Member_PostalCode": "90210",
-    "Member_Country": "USA"
+{
+    "first_name": "无",
+    "last_name": "名氏",
+    "plan_type": "GOLD",
+    "Member_Busname": "Makana Health",
+    "Member_Addr": "1 Salesforce Way",
+    "Member_City": "San Francisco",
+    "Member_State": "CA",
+    "Member_PostalCode": "90210",
+    "Member_Country": "USA",
+    "Member_Language": "CN"

```

Comment:
I would prefer if the default example remains using "en" as I feel that's going to be the largest audience.

Please also update the first/last name as well to accommodate that.

Reply:
Fixed in latest commit. I used a fictitious name (one of the Salesforcelandians).